### PR TITLE
Karkakol/add active simulcast encodings

### PR DESCRIPTION
**Pull request by karkakol**
## Description
Enhance types and models to handle active simulcast encodings, and export some functions.

## Motivation and Context
Allow user to change targetTrackEncoding, display that information

## How has this been tested?
Manual tests + e2e, tests result is pasted at the end.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

Running: app-release.apk on Android

» /test/specs/test.e2e.ts
Walk through app
   ✓ create room and peer to obtain credentials
   ✓ type jellyfish url and token
   ✓ request necessary permissions and connect
   ✓ toggle off preview camera and microphone then join the room
   ✓ check if no camera view
   ✓ toggle camera on
   ✓ check if only one video cell
   ✓ switch camera
   ✓ screen share on
   ✓ check if two video cells
   ✓ toggle camera off
   ✓ check if only 1 video cell
   ✓ screen share off
   ✓ check if no camera view again
   ✓ toggle microphone on
   ✓ toggle microphone off
   ✓ disconnect from room

17 passing (17.6s)
------------------------------------------------------------------
Running: JellyfishExample.ipa on iOS

» /test/specs/test.e2e.ts
Walk through app
   ✓ create room and peer to obtain credentials
   ✓ type jellyfish url and token
   ✓ request necessary permissions and connect
   ✓ toggle off preview camera and microphone then join the room
   ✓ check if no camera view
   ✓ toggle camera on
   ✓ check if only one video cell
   ✓ switch camera
   ✓ screen share on
   ✓ check if two video cells
   ✓ toggle camera off
   ✓ check if only 1 video cell
   ✓ screen share off
   ✓ check if no camera view again
   ✓ toggle microphone on
   ✓ toggle microphone off
   ✓ disconnect from room

17 passing (40.1s)

Spec Files:      2 passed, 2 total (100% completed) in 00:00:49

### CODE REVIEW

File: example/ios/Podfile [hunk 1]
```diff
@@ -85,7 +86,10 @@ target 'JellyfishExample' do
     # An absolute path to your application root.
     :app_path => "#{Pod::Config.instance.installation_root}/.."
   )
-  pod 'MembraneRTC'
+#   pod 'MembraneRTC'
```
Comment (incubo4u) [inline example/ios/Podfile:89]: change it back 🌻

File: src/JellyfishClientContext.tsx [hunk 2]
```diff
@@ -63,9 +63,12 @@ const JellyfishContextProvider = (props: any) => {
     if (websocket.current?.readyState !== WebSocket.OPEN) {
       return;
     }
+
     const message = PeerMessage.encode({
       mediaEvent: { data: event },
     }).finish();
+    console.log('send: ', { mediaEvent: { data: event } });
```
Comment (incubo4u) [inline src/JellyfishClientContext.tsx:70]: 👀

Comment (karkakol) [inline src/JellyfishClientContext.tsx:70]: removed

File: src/JellyfishClientContext.tsx [hunk 3]
```diff
@@ -102,6 +105,8 @@ const JellyfishContextProvider = (props: any) => {
         const uint8Array = new Uint8Array(event.data);
         try {
           const data = PeerMessage.decode(uint8Array);
+          console.log('Recive: ', data);
```
Comment (karkakol) [inline src/JellyfishClientContext.tsx:108]: removed

File: example/ios/Podfile (hunk 1 above)
Comment (karkakol) [inline example/ios/Podfile:89]: removed comment

File: example/ios/Podfile [hunk 4]
```diff
@@ -105,5 +109,7 @@ target 'JellyfishExample' do
 end
 
 target 'ScreenBroadcast' do
-  pod 'MembraneRTC/Broadcast'
+  pod 'MembraneRTC/Broadcast', :git => 'https://github.com/jellyfish-dev/membrane-webrtc-ios.git', :branch => 'master'
```
Comment (karkakol) [inline example/ios/Podfile:112]: correct file source

**Pull request merged**